packages/edit-post/src/components/layout/index.js

```diff
@@ -6,163 +6,142 @@ import classnames from 'classnames';
+			isSaving: select( 'core/edit-post' ).isSavingMetaBoxes(),
+		} );
+	} );
 	const sidebarIsOpened = editorSidebarOpened || pluginSidebarOpened || publishSidebarOpened;
-
 	const className = classnames( 'edit-post-layout', 'is-mode-' + mode, {
 		'is-sidebar-opened': sidebarIsOpened,
 		'has-fixed-toolbar': hasFixedToolbar,
 		'has-metaboxes': hasActiveMetaboxes,
 	} );
 
-	const publishLandmarkProps = {
-		role: 'region',
-		/* translators: accessibility text for the publish landmark region. */
-		'aria-label': __( 'Editor publish' ),
-		tabIndex: -1,
-	};
 	return (
-		<FocusReturnProvider className={ className }>
+		<>
 			<FullscreenMode />
 			<BrowserURL />
 			<UnsavedChangesWarning />
 			<AutosaveMonitor />
 			<LocalAutosaveMonitor />
-			<Header />
-			<div
-				className="edit-post-layout__content edit-post-layout__scrollable-container"
-				role="region"
-				/* translators: accessibility text for the content landmark region. */
-				aria-label={ __( 'Editor content' ) }
-				tabIndex="-1"
-			>
-				<EditorNotices />
-				<EditorModeKeyboardShortcuts />
-				<KeyboardShortcutHelpModal />
-				<ManageBlocksModal />
-				<OptionsModal />
-				{ ( mode === 'text' || ! isRichEditingEnabled ) && <TextEditor /> }
-				{ isRichEditingEnabled && mode === 'visual' && <VisualEditor /> }
-				<div className="edit-post-layout__metaboxes">
-					<MetaBoxes location="normal" />
-				</div>
-				<div className="edit-post-layout__metaboxes">
-					<MetaBoxes location="advanced" />
-				</div>
-				{ isMobileViewport && sidebarIsOpened && <ScrollLock /> }
-			</div>
-			{ isRichEditingEnabled && mode === 'visual' && (
-				<div
-					className="edit-post-layout__footer"
-					role="region"
-					/* translators: accessibility text for the content landmark region. */
-					aria-label={ __( 'Editor footer' ) }
-					tabIndex="-1"
-				>
-					<BlockBreadcrumb />
-				</div>
-			) }
-			{ publishSidebarOpened ? (
-				<PostPublishPanel
-					{ ...publishLandmarkProps }
-					onClose={ closePublishSidebar }
-					forceIsDirty={ hasActiveMetaboxes }
-					forceIsSaving={ isSaving }
-					PrePublishExtension={ PluginPrePublishPanel.Slot }
-					PostPublishExtension={ PluginPostPublishPanel.Slot }
-				/>
-			) : (
-				<>
-					<div className="edit-post-toggle-publish-panel" { ...publishLandmarkProps }>
-						<Button
-							isDefault
-							type="button"
-							className="edit-post-toggle-publish-panel__button"
-							onClick={ togglePublishSidebar }
-							aria-expanded={ false }
-						>
-							{ __( 'Open publish panel' ) }
-						</Button>
-					</div>
-					<SettingsSidebar />
-					<Sidebar.Slot />
-				</>
-			) }
+			<EditorModeKeyboardShortcuts />
+			<ManageBlocksModal />
+			<OptionsModal />
+			<KeyboardShortcutHelpModal />
 			<Popover.Slot />
 			<PluginArea />
-		</FocusReturnProvider>
+			<FocusReturnProvider className={ className }>
+				<EditorRegions
+					className={ className }
```

Comment:
Is it necessary that `className` be applied on _both_ the `FocusReturnProvider` and `EditorRegions` elements?

Reply:
No, it's unintentional

Reply:
https://github.com/WordPress/gutenberg/pull/18967